WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsMenuViewController.swift

```diff
@@ -279,7 +283,8 @@ private extension InPersonPaymentsMenuViewController {
     }
 
     func configureOrderCardReader(cell: LeftImageTableViewCell) {
-        prepareForReuse(cell, accessibilityID: "order-card-reader")
+        prepareForReuse(cell)
+        cell.accessibilityIdentifier = "order-card-reader"
```

Comment:
Not a blocker: Should we move those identifiers into a Constants extension?

Reply:
Given that they're not used elsewhere, and are intrinsically descriptive, I don't see much reason to. It might be handy to do it when they're actually used in tests? We can do it later if so.